packages/wonder-blocks-popover/src/components/popover.tsx

```diff
@@ -101,6 +102,8 @@ type Props = AriaProps &
          * Whether to show the popover tail or not. Defaults to true.
          */
         showTail: boolean;
+
+        disablePortal?: boolean;
```

Comment:
**suggestion:** This could also be created in a different way... defining a  required `portal` (`true` by default) prop, then setting it as `false` when needed. This way, it would help us describing the need for this portal + explaining in which cases `false` makes sense. Another benefit is that the code could become a bit more explicit,  see the `maybeRenderPortal` example. What do you think?

related: Could you also please add comments to this prop so we can display it in Storybook? you might need to add that to `ArgTypes` as well (not fully sure about this part).

Reply:
Makes sense, updated the code and added a comment to the prop. Let me know what you think!